katran/lib/KatranLb.cpp

```diff
@@ -566,6 +567,7 @@ bool KatranLb::modifyRealsForVip(
       LOG(ERROR) << "Invalid real's address: " << real.address;
       continue;
     }
+    auto raddr = folly::IPAddress(real.address);
```

Comment:
on the last part - no it is not, by default network would fail validation, unless second param set to true (allow network to pass validation). line 566 does not do that